make/toolchain-arm32.cmake

```diff
@@ -48,9 +50,10 @@ endif ()
 SET(CMAKE_C_COMPILER   arm-${TARGET_ABI}-gcc)
 SET(CMAKE_CXX_COMPILER arm-${TARGET_ABI}-g++)
 
-# Assuming the cross compiler is installed to /usr/bin,
-# we do not need to set where the target environment is.
-SET(CMAKE_FIND_ROOT_PATH)
+# To build the tests, we need to set where the target environment containing
+# the required library is. On Debian-like systems, this is
+# /usr/arm-linux-gnueabihf/.
+SET(CMAKE_FIND_ROOT_PATH "/usr/arm-linux-gnueabihf")
```

Comment:
This should be parameterized with TARGET_ABI rather than hardcoded (I assume you hardcoded just to see what would work: normally we'd put a NOCHECKIN or sthg but the up-front NOCHECKIN CI checks make that practice no longer work -- maybe just a FIXME)